package.json

```diff
@@ -102,7 +102,9 @@
     "prepublishOnly": "npm run test",
```

Comment:
`prepublishOnly` should do then `npm run clean && npm run build:prod && npm run test`